blocks/header/__main-illustration/header__main-illustration.css

```diff
@@ -0,0 +1,12 @@
+
+.header__main-illustration {
+    margin:0;
+    padding:0;

```

Comment:
There's generally no need to reset margin and padding for elements like `img`, `div`, and `span` because these elements typically have default margin and padding values.

Please double-check your styles to re-evaluate whether any unnecessary adjustments can be removed.